lib/rust/ensogl/component/breadcrumbs/src/lib.rs

```diff
@@ -496,10 +522,31 @@ ensogl_core::define_endpoints_2! {
         move_up(),
         /// Move the selection to the next (lower-level) breadcrumb in the list.
         move_down(),
+        /// Remove the last breadcrumb from the list.
+        pop(),
+        /// Remove the last `n` breadcrumbs from the list.
+        pop_multiple(usize),
+        /// Remove the last `n` breadcrumbs from the list, but only up to the first `m` breadcrumbs.
+        pop_multiple_but_retain((usize,usize)),
+
+        // == Theming API ==
+
+        /// Set the color of the text.
+        set_text_selected_color(color::Rgba),
+        /// Set the color of the text for greyed out breadcrumbs.
+        set_text_greyed_out_color(color::Rgba),
+        /// Set the color of the separator icon.
+        set_background_color(color::Rgba),
+        /// Set the color of the background.
+        set_separator_color(color::Rgba),
     }
     Output {
         /// Currently selected breadcrumb.
-        selected(BreadcrumbId)
+        selected(BreadcrumbId),
+        /// Only emitted when the selection is changed through user interaction.
+        user_selected(BreadcrumbId),
```

Comment:
Firstly, I think it's better to say that it's emitted when the selection is changed by clicking on the breadcrumb. After all, entering/leaving nodes is also a `user interaction`, but I assume it will not be emitted then.
Secondly, I don't see any usage of it in the code.

Reply:
This was actually not implemented/needed in the end, so it is removed now.